template/src/app/signed-out/SignIn.tsx

```diff
@@ -51,7 +51,7 @@ function SignIn() {
         />
 
         {Platform.OS !== 'web' && <Facebook />}
-        {Platform.OS !== 'web' && <Google />}
+        <Google />
```

Comment:
Just want to say that everything else is important to make this happen, but this: this is the cool part :sunglasses: